src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/HandshakeProtocol.cs

```diff
@@ -58,7 +58,8 @@ private static JsonTextWriter CreateJsonTextWriter(Stream output)
 
         private static JsonTextReader CreateJsonTextReader(ReadOnlyMemory<byte> payload)
         {
-            var textReader = new Utf8BufferTextReader(payload);
+            var textReader = new Utf8BufferTextReader();
```

Comment:
Why doesn't this place use `Utf8BufferTextReader.Get(payload);`?

Reply:
I didn't want to change the pattern. This creates the JsonTextReader, I want to put a try finally around the whole operation that does the return. I'll fix it up in another PR.